### DISCUSSION TITLE
Microsoft store: App renamed to RESP.app?

### DISCUSSION THREAD

**Discussion by tony** [redis/RedisDesktopManager#5170] [Q&A] [answered]
Hi!  Thank you for the very excellent application

Is there any more context on the app's name being changed?

In re: https://www.microsoft.com/en-us/p/respapp-developer-gui-for-redis-formerly-redisdesktopmanager/9ndk76zvz3tm

![image](https://user-images.githubusercontent.com/26336/147883911-725adb3a-f9c8-4c9e-b1a2-f6b31a4720b3.png)

Was this a trademark issue? Something relating to Microsoft Store's rules?

Thank you!

**Comment by uglide** [comment, marked as answer]
Hello,

Thank you for your concern. I will quote here my announcement from Telegram chat to spill more light on this renaming:

<blockquote>
Igor, [31 Dec 2021, 10:29:35]:
...Happy holidays! I want to make an announcement to clarify what is happening with the project.

Back in 2019, I was contacted by lawyers of RedisLabs which requested to stop violating/using their trademark “Redis” in the name RedisDesktopManager. So the project name was shortened to just RDM which is obviously an acronym of the original name. Back then it looked like a good solution but it appeared that RDM is a trademark of another company which is now demanding to stop violating their rights. This project is not a competitor to theirs and to be honest I never heard of them, but I do not have the willines or resources to be stuck in lawsuits for years.

So … In 2022 I’m planning to rename this project.

Many users are confused about those three letters (RDM) and don’t really understand the meaning and connection to Redis. So I started to brainstorm some new naming concepts for this project and came up with RESP app. Resp stands for REdis Serialization Protocol which is used for client-server communication in Redis. So, the naming RESP app clearly describes the purpose of this project and my true and forever focus on Redis.

I heard all your feedback about logo shape on macOS and dark mode issues. So 2022.0 will be released with a new 2 color icon set to make it not so bright in dark mode. Plus it will include app layout customizations which allow you to improve productivity when you inspect data in Redis.
</blockquote>

So, basically, it's a chain of trademark issues which is hopefully will be broken by this project renaming.

Regards,

**Reply by tony** [reply]
@uglide Thank you for the explanation on this.
